arm-apimanagement/2017-03-01/swagger/apimtenant.json

```diff
@@ -0,0 +1,736 @@
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/tenant/{accessName}": {
+      "get": {
+        "tags": [
+          "TenantAccess"
+        ],
+        "operationId": "TenantAccess_Get",
+        "description": "Get tenant access information details.",
+        "x-ms-examples": {
+          "ApiManagementGetTenantAccess": {
+            "$ref": "../examples/ApiManagementGetTenantAccess.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/AccessParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Tenant Access information.",
+            "schema": {
+              "$ref": "#/definitions/AccessInformationContract"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "TenantAccess"
+        ],
+        "operationId": "TenantAccess_Update",
+        "description": "Update tenant access information details.",
+        "x-ms-examples": {
+          "ApiManagementUpdateTenantAccess": {
+            "$ref": "../examples/ApiManagementUpdateTenantAccess.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AccessInformationUpdateParameters"
+            },
+            "description": "Parameters."
```

Comment:
`"description" : "Parameters"` - Description must not match the name of the node it is supposed to describe.